nixos/maintainers/scripts/lxd/lxd-virtual-machine-image-inner.nix

```diff
@@ -0,0 +1,95 @@
+# Edit this configuration file to define what should be installed on
+# your system.  Help is available in the configuration.nix(5) man page
+# and in the NixOS manual (accessible by running ‘nixos-help’).
```

Comment:
Would you mind removing the comments below. I believe it should be enough with the reference to `configuration.nix(5)`

Reply:
Yep, I've stripped the bulk of the comments from the two image inner files.